asistente_ladm_col/tests/test_quality_validations.py

```diff
@@ -569,7 +569,33 @@ def test_check_gaps_in_plots(self):
 
         test_plots_layer.rollBack()
 
+    def test_multiparts_in_right_of_way(self):
+        print('\nINFO: Validating boundary_dangles with no dangles...')
```

Comment:
Change info message

Reply:
Changed to: Validating right_of_way for no multipart geometries